MediaRecorder.html

```diff
@@ -182,11 +182,11 @@
 <section id="blob-event">
     <h2>Blob Event</h2>
     <dl title='interface BlobEvent : Event' class='idl'>
-        <dt>Constructor(DOMString type, BlobEventInit eventInit)</dt>
+        <dt>Constructor(DOMString type, required BlobEventInit eventInitDict)</dt>
```

Comment:
I just found out too. Moving the |required| to BlobEventInit.data in https://github.com/w3c/mediacapture-record/pull/39